### ISSUE TITLE
Utilities.IsValid(Networking.LocalPlayer) returns false

### ISSUE THREAD

**Issue by Guribo** [CyanLaser/CyanEmu#33]
This is not happening in VRChat but only in the Unity editor.

It can easily be tested with a simple U# script containing e.g. the following:

```
        private void Update()
        {
            if (Networking.LocalPlayer != null)
            {
                Debug.Log("Local player is not null");
            }
            
            if (!Utilities.IsValid(Networking.LocalPlayer))
            {
                Debug.LogError("Local player is not valid");
            }
        }
```

Result in editor:  
![image](https://user-images.githubusercontent.com/49484893/111067549-26d74d00-84c5-11eb-953c-501172d4feae.png)

Result in game:  
![image](https://user-images.githubusercontent.com/49484893/111067791-1d9ab000-84c6-11eb-8b8e-1235c537d919.png)

Tested with Cyan Emu v0.3.4 Beta and Udon Sharp v0.19.5.

**Comment by CyanLaser** [comment]
Thank you for finding this. I'll look into it. Do you know of any other items that might have improper IsValid implementation?

**Comment by Guribo** [comment]
So far I only noticed it for VRCPlayerApis, I will let you know when I come accross others.

**Comment by CyanLaser** [comment]
Hopefully fixed in e53d01f568698cb2da703969f24c4cd83e8a8990

**Comment by Guribo** [comment]
It is working now. Thanks!
